src/services/followers.service.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  addOneFollower,
+  deleteFollowerById,
+  listFollowers,
+} from "../repositories/followers.repository";
+
+export const getFollowers = async (query: object) => {
+  const followersList = await listFollowers(query);
+  return followersList;
+};
+
+export const addFollower = async (ownerId: string, followerId: string) => {
```

Comment:
```suggestion
export const addFollower =  (ownerId: string, followerId: string) => {
```
not use async without await